bl/blmain.c

```diff
@@ -46,6 +46,7 @@ uint32_t bl_adc_random_seed(void);
 int main(void) {
     __disable_irq();
     clk_setup_pll();
+    LL_APB1_GRP1_EnableClock(LL_APB1_GRP1_PERIPH_USART2);
```

Comment:
I think this should be #if USART_IDX==2 ... like in bluart.c

Reply:
ok, will amend